packages/lesswrong/server/apolloServer.ts

```diff
@@ -201,12 +201,12 @@ export function startWebserver() {
     }
     
     const currentUser = await getUserFromReq(req)
-    if (!currentUser || !getUserEmail(currentUser)){
+    if (!currentUser || !currentUser.email){
```

Comment:
This... does not seem like a safe substitution?

Reply:
I thought I put that back as it was